client/src/pages/dashboard/Dashboard.tsx

```diff
@@ -8,14 +8,44 @@ import { useUser } from '../../hooks/useUser';
 import getAuthErrorMessage from '../../utils/getAuthErrorMessage';
 import { showToast } from '../../utils/showToast';
 import Modal from '../../ui/Modal/Modal';
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+import Avatar from '@mui/material/Avatar';
+import { red } from '@mui/material/colors';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import IconButton, { IconButtonProps } from '@mui/material/IconButton';
+import { CardContent, Menu, MenuItem } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
```

Comment:
Move all import to new component, remove IconButtonProps and Avatar because it's not needed. Think about colors, are we using our theme or mui? Let's discuss.

Reply:
Okej. I must check colors. I need more time to do this